fontbe/src/error.rs

```diff
@@ -11,4 +13,45 @@ pub enum Error {
     FeaAssembleError(#[from] BinaryCompilationError),
     #[error("Fea compilation failure")]
     FeaCompileError(#[from] CompilerError),
+    #[error("'{0}' {1}")]
+    GlyphError(GlyphName, GlyphProblem),
+    #[error("'{0}' {1:?} {2}")]
+    KurboError(GlyphName, BadKurbo, String),
```

Comment:
My personal rule of thumb is I only use tuple-structs if there is a single member, and I would definitely switch to named fields when there are 3+ members.

Reply:
Oh good catch, will fix. My rule of thumb is if it's not obvious from the types and that String is not obvious. If it was GlyphName BadKurbo SomeInformativeTYpe I'd feel better.